packages/bot-web-ui/src/components/dashboard/tutorial-components/index.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import Tutorial from './tutorial';
+
+const Index = () => {
+    return <Tutorial />;
+};
+
+export default Index;
```

Comment:
I think we can just export this directly 

```
export { Tutorial } 
```

Reply:
Ok